src/Microsoft.DotNet.Build.CloudTestTasks/CreateAzureContainer.cs

```diff
@@ -77,6 +77,8 @@ public override bool Execute()
 
         public async Task<bool> ExecuteAsync()
         {
+            bool success = false;
```

Comment:
I'd rather we just "return false" when we get an error. This Boolean leads to some strange code paths where we've "failed", but keep running anyway. (And is also adding to the indent level in some cases).